src/realm/object-store/shared_realm.cpp

```diff
@@ -90,6 +90,16 @@ Realm::Realm(Config config, util::Optional<VersionID> version, std::shared_ptr<_
     m_coordinator = std::move(coordinator);
 }
 
+Realm::Realm(const Realm& other, MakeFrozenTag)
+    : m_coordinator(other.m_coordinator)
+    , m_config(other.m_config)
```

Comment:
I'm not sure this will work as intended - we need to replace the config scheduler to use the frozen one, otherwise Core will prevent us from moving between threads.

Reply:
I've been convinced 👍